packages/eslint-plugin-qwik/src/validLexicalScope.ts

```diff
@@ -486,7 +489,6 @@ const ALLOWED_CLASSES = {
   Set: true,
   Map: true,
   Uint8Array: true,
-  JSXNodeImpl: true,
```

Comment:
why do you remove this?

Reply:
I think the change now covers such the case.
I had added the `JSXNodeImpl` to the `ALLOWED_CLASSES` in order to fix the issue previously here:
https://github.com/QwikDev/qwik/pull/5883
But now I found the right answer to do this, so I have deleted it.